change/@fluentui-react-rating-deec31e1-3da3-442d-a5f3-dc7a74057fd0.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31034&commitId=dd2d6ec837c9b6c10d0f6e842a7d9c14a9b21bbb&env=prod&clientType=FLUENTUI)


  <details><summary><b>RatingDisplay Converged</b> 18 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|RatingDisplay Converged.Neutral with half value.rest.chromium.png|60| Changed|
|RatingDisplay Converged.Compact.rest.chromium.png|31| Changed|
|RatingDisplay Converged.rounded down.rest.chromium.png|59| Changed|
|RatingDisplay Converged.custom circle icons.rest.chromium.png|31| Changed|
|RatingDisplay Converged.Brand with half value - High Contrast.rest.chromium.png|70| Changed|
|RatingDisplay Converged.size large with value and count.rest.chromium.png|166| Changed|
|RatingDisplay Converged.Brand with half value.rest.chromium.png|60| Changed|
|RatingDisplay Converged.size medium with value and count.rest.chromium.png|124| Changed|
|RatingDisplay Converged.custom square icons.rest.chromium.png|31| Changed|
|RatingDisplay Converged.rounded up.rest.chromium.png|70| Changed|
|RatingDisplay Converged.Marigold with half value - Dark Mode.rest.chromium.png|59| Changed|
|RatingDisplay Converged.Brand with half value - Dark Mode.rest.chromium.png|59| Changed|
|RatingDisplay Converged.Marigold with half value - High Contrast.rest.chromium.png|70| Changed|
|RatingDisplay Converged.size extra-large with value and count.rest.chromium.png|261| Changed|
|RatingDisplay Converged.Marigold with half value.rest.chromium.png|60| Changed|
|RatingDisplay Converged.Neutral with half value - Dark Mode.rest.chromium.png|59| Changed|
|RatingDisplay Converged.Neutral with half value - High Contrast.rest.chromium.png|70| Changed|
|RatingDisplay Converged.size small with value and count.rest.chromium.png|124| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>